conference.js

```diff
@@ -1193,6 +1193,10 @@ export default {
                 (typeof config.hosts.jirecon != "undefined"))?
                 "jirecon" : "colibri";
         }
+        let nick = APP.settings.getDisplayName();
+        if (nick) {
+            options.displayName = nick;
```

Comment:
Is it necessary to modify the global config object? Is the case where the display name changes not important for now?

Reply:
Yep, we need the value if any when creating the conference object. This was the only way to do that without modifying the lib-jitis-meet API. And this is done in _getConferenceOptions which seems like the right place. Maybe we need to clone config here and modify the cloned object, is this your proposal?

Reply:
I think in general modifying a global hash is tricky and not ideal, just as a principle in case something somewhere else assumed that global value exists, thereby creating a hidden temporal dependency. I don't propose you do it now because I just noticed the logic above already modifies the global.

I am curious about the case where the display name gets changed. Is that not a problem for now?

Reply:
Well, we can send it when initializing callstats. So if it changes ... it will use only the initial one, and re-initializing callstats is not a good idea I think. So for now it is ok.